packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3064,6 +3064,9 @@ export class ContainerRuntime
 				this.disableAttachReorder !== true
 			) {
 				this.outbox.submitAttach(message);
+			} else if (type === ContainerMessageType.BlobAttach) {
+				// BlobAttach ops must have their metadata visible and cannot be grouped (see opGroupingManager.ts)
+				this.outbox.submitBlobAttach(message);
```

Comment:
Is there any way for this kind of decision (type === ContainerMessageType.BlobAttach) to be next to where op is generated? I think ContainerMessageType.Attach case will go away one way or another over time, and it would be great for this code not to have to encode some knowledge about specific ops. I'd rather add boolean argument to this function that says something like "this op can't be batched" vs. having this knowledge here.